cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -120,4 +89,81 @@ describe('Creator Settings Page', () => {
     cy.findByRole('button', { name: 'Finish' }).click();
     cy.url().should('equal', `${baseUrl}admin/creator_settings/new`);
   });
+
+  context('color contrast ratios', () => {
+    it('should show an error when the constrast ratio of a brand color is too low', () => {
+      const lowContrastColor = '#a6e8a6';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(lowContrastColor)
+        .trigger('change');
+
+      cy.findByText(
+        /^The selected color must be darker for accessibility purposes./,
+      ).should('be.visible');
+    });
+
+    it('should not show an error when the constrast ratio of a brand color is good', () => {
+      const adequateContrastColor = '#25544b';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(adequateContrastColor)
+        .trigger('change');
+
+      cy.findByText(
+        /^The selected color must be darker for accessibility purposes./,
+      ).should('not.be.visible');
+    });
+  });
+
+  context('brand color updates', () => {
+    it('should not update the brand color if the color contrast ratio is low', () => {
+      const lowContrastColor = '#25544b';
+      const lowContrastRgbColor = 'rgb(37, 84, 75)';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(lowContrastColor)
+        .trigger('change');
+
+      cy.findByRole('button', { name: 'Finish' }).should(
+        'not.have.css',
+        'background-color',
+        lowContrastRgbColor,
+      );
```

Comment:
I think this test would pass even if the behaviour were incorrect, because the color change is async, and cypress would be able to validate 'color same as before change' earlier than it's able to validate the 'color different after change'.

I think to make this more robust, we should first validate that the error message has also appeared (since it should appear at the same time as any color change takes place).